sdks/java/core/src/main/java/org/apache/beam/sdk/options/PipelineOptions.java

```diff
@@ -353,4 +354,38 @@ public Long create(PipelineOptions options) {
       return NEXT_ID.getAndIncrement();
     }
   }
+
+  /**
+   * A user agent string as per RFC2616, describing the pipeline to external services.
+   *
+   * <p>https://www.ietf.org/rfc/rfc2616.txt
+   *
+   * <p>It should follow the BNF Form:
+   * <pre><code>
+   * user agent         = 1*(product | comment)
+   * product            = token ["/" product-version]
+   * product-version    = token
+   * </code></pre>
+   * Where a token is a series of without a separator.
```

Comment:
Awkward sentence: `Where a token is a series of without a separator.`

Reply:
Whoops, forgot a word.